src/obfl-specification.html

```diff
@@ -2925,15 +2962,81 @@ <h4 id="L2475">pre-translated</h4>
 
 <p>Note that:</p>
 <ul>
-  <li>BRAILLE PATTERN BLANK (U+2800) is not whitespace nor line breaking. It
-    can be used in place of NO-BREAK SPACE (U+00A0). For other purposes, use
-    U+0020</li>
+  <li>BRAILLE PATTERN BLANK (U+2800) is not regarded as white space and should
+    be preserved. Proper white space characters must be used at word boundaries
+    in order to achieve correct line breaking.</li>
   <li>pre-translated text nodes are not influenced by the hyphenate attribute
     nor the global hyphenation policy</li>
   <li class="markup">style elements MUST NOT occur in pre-translated
   context</li>
 </ul>
 
+<!--
```

Comment:
Please indicate why this is commented out. E.g. "TO DO"

Reply:
I had commented it out because it is a value of the `TranslatorType` enum (in the `TranslatorMode` class), but I fail to see what it is for and why one would want to use this instead of "pre-translated". For now I removed the comment...